exdb/forms.py

```diff
@@ -83,9 +84,12 @@ def __init__(self, *args, **kwargs):
 class ExperienceSubmitForm(ExperienceSaveForm):
 
     def clean(self):
-        ex_subtype = self.cleaned_data.get('subtype')
-        needs_verification = ex_subtype.needs_verification if ex_subtype else None
-        name = "" if not ex_subtype else ex_subtype.name
+        ex_subtype = self.cleaned_data.get('subtypes')
+        needs_verification = False
```

Comment:
If no subtypes are selected, this form erroneously says that the conclusion & attendance fields need to be filled out. To avoid that, we should either default `needs_verification` to True or [copy the logic we used in the javascript](https://github.com/mostateresnet/exdbproject/blob/6debc394279232c555660fa41c2a848be33dd27d/exdb/static/exdb/js/experience_form.js#L7).